### PR TITLE
add new openbabel package

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: recipes/openbabel/2.3.2_7d621d9/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  build:
+    - cmake
+    - zlib
+    - swig
+    - eigen
+    - bzip2
+    - python
+    - zlib
+    - libxml2
```
Comment (johanneskoester) [inline recipes/openbabel/2.3.2_7d621d9/meta.yaml]: Do you want to add gcc/llvm here?

Comment (bgruening) [inline recipes/openbabel/2.3.2_7d621d9/meta.yaml]: Ah good point ... can we add a check to travis for this? Some kind of linting?

Comment (johanneskoester) [inline recipes/openbabel/2.3.2_7d621d9/meta.yaml]: That's a great idea. One way is to remove gcc and llvm from the docker container.

Comment (bgruening) [inline recipes/openbabel/2.3.2_7d621d9/meta.yaml]: Ok, let's do this and zlib as well :)

Comment (bgruening) [inline recipes/openbabel/2.3.2_7d621d9/meta.yaml]: @johanneskoester https://github.com/bioconda/bioconda-recipes/pull/1201 ;)

Comment (johanneskoester) [inline recipes/openbabel/2.3.2_7d621d9/meta.yaml]: Thanks!

**Pull request merged**